packages/markers/marker.go

```diff
@@ -67,7 +67,7 @@ func (m *Marker) Index() (index Index) {
 }
 
 // Bytes returns a marshaled version of the Marker.
-func (m *Marker) Bytes() (marshaledMarker []byte) {
+func (m Marker) Bytes() (marshaledMarker []byte) {
```

Comment:
Why make it a `Marker` receiver?

Reply:
I think it was needed because a `Marker` is used in the thresholdevent as a map key.